modules/command/dv/dv-purchase-initiate-command.js

```diff
@@ -137,9 +138,10 @@ class DvPurchaseInitiateCommand extends Command {
         }, { where: { handler_id } });
     }
 
-    async _validatePrivateDataRootHash(dataSetId, otObjectId, private_data_root_hash) {
-        const privateObject = await this.importService.getPrivateDataObject(dataSetId, otObjectId);
-        return private_data_root_hash === privateObject.private_data_hash;
+    async _validatePermissionedDataRootHash(dataSetId, otObjectId, permissioned_data_root_hash) {
```

Comment:
This method can be places in permissioned-data-service, i think that we will need this code somewhere else, if not now, maybe in a future.